package.json

```diff
@@ -50,7 +50,7 @@
     "glob": "7.1.6",
     "graphql": "14.5.8",
     "graphql-tag": "2.10.1",
-    "helmet": "3.21.2",
+    "helmet": "^3.21.2",
```

Comment:
Should not have any change, especially because we want to use fixed dependencies